### PR TITLE
[alerts] Add OpenSearch metric and alert

### PR DESCRIPTION
**Pull request by zhquan**
This commit adds OpenSearch server metric and absent alert.

### CODE REVIEW

File: terraform/modules/bap_alerting_gcp/opensearch.tf [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Basic OpenSearch metrics and alerts
+
+resource "google_logging_metric" "opensearch_server_count" {
+  name        = "opensearch.server.count"
+  description = "Number of OpenSearch server"
```
Comment (sduenas) [inline terraform/modules/bap_alerting_gcp/opensearch.tf:5]:
```suggestion
  description = "Number of OpenSearch active servers"
```

File: terraform/modules/bap_alerting_gcp/opensearch.tf [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# }
+
+# resource "google_logging_metric" "opensearch_server_started" {
+#   name        = "opensearch.server.started"
+#   description = "When OpenSearch server receive the started message"
+
+#   count         = var.alerts ? 1 : 0
+
+#   filter      = "logName : \"opensearch-logs$\" AND jsonPayload.message: \"started\" AND jsonPayload.level: (\"INFO\")"
+# }
```
Comment (sduenas) [inline terraform/modules/bap_alerting_gcp/opensearch.tf:28]: Why are these commented?

**Review by sduenas — CHANGES_REQUESTED** [review]
It looks good but we need to clarify what we do with the other metrics you defined that are commented.

File: terraform/modules/bap_alerting_gcp/opensearch.tf (hunk 2 above)
Comment (zhquan) [reply terraform/modules/bap_alerting_gcp/opensearch.tf]: These could be good metrics but we don't have alerts for these. I remove them

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**